runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +97,19 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight() && ((plugin.getInventoryItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getInventoryItems()) || (plugin.getEquippedItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getEquippedItems())))))
+		{
+			for (Varbits varbit: BRAZIER_VARBITS)
+			{
+				if (client.getVar(varbit) == 1)
+				{
+					break;
+				}
+			}
```

Comment:
The braziers only affect a certain area, so you should only check the ones that a clue would need